### ISSUE TITLE
Graphviz generator crashes on `sizeof<>` expressions

### ISSUE THREAD

**Issue by jtbandes** [kaitai-io/kaitai_struct#966]
The following definition crashes the graphviz generator:

```yaml
meta:
  id: asdf
  file-extension: asdf

seq:
  - id: x
    size: sizeof<xx>
    
types:
  xx:
    seq:
      - id: a
        type: u1
```

I believe the problem is that [this `match`](https://github.com/kaitai-io/kaitai_struct_compiler/blob/0acfa60452a10b770a2c2e746f3de057f8ab76d9/shared/src/main/scala/io/kaitai/struct/GraphvizClassCompiler.scala#L282) doesn't handle sizeofs.

This is preventing `log/mcap.ksy` from having a svg generated at https://formats.kaitai.io/mcap/

**Comment by jtbandes** [comment]
@generalmimon Thanks for fixing this! I still see a crash when I test it out on ide.kaitai.io, but guessing this just means the new version hasn't been deployed yet. Looking forward to it :)

**Comment by generalmimon** [comment]
@jtbandes:

> I still see a crash when I test it out on ide.kaitai.io, but guessing this just means the new version hasn't been deployed yet.

Correct. As noted at https://doc.kaitai.io/user_guide.html#web-ide, https://ide.kaitai.io/ hosts the latest "stable" Web IDE and compiler version, so it uses the released KSC 0.10.

https://ide.kaitai.io/devel/ is _supposed to_ have the latest development version of KSC, but unfortunately it has a half-year old compiler at this point. In the past, [there used to be a script](https://github.com/kaitai-io/kaitai_struct/issues/573#issuecomment-575998459) that automatically triggered a rebuild of the Web IDE when a new version of KSC was available, but it was written for Travis CI and we migrated to GitHub Actions since then (see https://github.com/kaitai-io/kaitai_struct_webide/commit/0a79a8e60f8748fb46f3022a48b9a495ce97c352), but we [haven't migrated the script](https://github.com/kaitai-io/kaitai_struct/pull/1044#pullrequestreview-1541917941) that would trigger the Web IDE rebuild yet. I want to get around to it someday, but I don't know when that will be.

In the meantime, you can build the compiler [from source](https://doc.kaitai.io/serialization.html#_building_the_compiler_from_source) (but checkout the default `master` branch, not <code><del>serialization</del></code>) to try out the development version of KSC before it is released.

**Comment by jtbandes** [comment]
Here's a proposal for how to migrate the auto-rebuild behavior:

https://github.com/kaitai-io/kaitai_struct_webide/pull/181
https://github.com/kaitai-io/kaitai_struct/pull/1124
